bwakit/Dockerfile

```diff
@@ -2,16 +2,20 @@ FROM ubuntu
 
 MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
 
-RUN apt-get update && \
-  apt-get install -y \
-    curl
+RUN apt-get update; apt-get install -y wget bzip2 perl
 
 # pull down bwa-kit
-RUN curl -L http://sourceforge.net/projects/bio-bwa/files/bwakit/bwakit-0.7.12_x64-linux.tar.bz2 \
-  | tar -xjC /opt/
+RUN wget http://sourceforge.net/projects/bio-bwa/files/bwakit/bwakit-0.7.12_x64-linux.tar.bz2
```

Comment:
I liked the streaming extraction better. What's your reason for changing it? This new way is leaking the tarball and is more verbose as it duplicates the file name. The only thing I would change is to indent `|` to align with `c` in `curl`.